packages/storybook/src/nimble/tabs/tabs.stories.ts

```diff
@@ -42,18 +44,58 @@ const metadata: Meta<TabsArgs> = {
     }
 };
 
+const simpleTabs: readonly TabArgs[] = [
+    { title: 'Tab 1', id: '0', disabled: false },
+    { title: 'Tab 2', id: '1', disabled: true },
+    { title: 'Tab 3', id: '2', disabled: false }
```

Comment:
The id vs the index being off is really confusing. i.e. selecting tab 1 says "content of tab 0":
![image](https://github.com/user-attachments/assets/54072e9e-aba1-42dd-9ec1-ae0e21bdea87)